packages/integrations/image/README.md

```diff
@@ -82,6 +82,14 @@ Or, alternatively if your project is using the types through a `tsconfig.json`
 
 ## Usage
 
+__`index.astro`__
+
+```astro
+---
+import { Image, Picture } from '@astrojs/image/components';
+---
+```
```

Comment:
``````suggestion
```astro title="src/pages/index.astro"
---
import { Image, Picture } from '@astrojs/image/components';
---
```
``````
Just added our special docs syntax so we get a nice Hippo tag in docs! But, I totally agree with putting this exactly right here!